### ISSUE TITLE
v1 tag not released to github marketplace

### ISSUE THREAD

**Issue by davidkhala** [softprops/action-gh-release#231]
As shown in https://github.com/marketplace/actions/gh-release

![image](https://user-images.githubusercontent.com/7227589/168088686-1f4e064c-f8f6-40b5-a1ed-816bcf7683e9.png)
The latest available here is v0, instead of v1 in README. Any plan to release v1?

**Comment by briantist** [comment]
@davidkhala that error is not from this action, it's a setting in your repository or your organization that limits which actions can run. You can see this in a repository by browsing to `Settings > Actions > General`

![image](https://user-images.githubusercontent.com/1260690/169676122-e3486f8e-abe5-4939-99f9-13439fd08cb6.png)

**Comment by davidkhala** [comment]
@briantist yes, I see this as a workaround -- to add this action into white list. But I wonder why softprops is not a verified creator

**Comment by briantist** [comment]
Well, it's up to the author if they want to try to get verified; I can't answer for that. 

But if that's your question, you should update the title and body of the issue.

The error message you're getting is a setting you need to change. And as for the title, showing up the marketplace is not related to what you can use after the `@`. Any commit-ish works (a tag, a branch, a commit SHA), and there's no separate process to publish specific ones (an action also doesn't need to be on the marketplace to be used).

**Comment by mikepenz** [comment]
The conversation on becoming a verified creator is sadly much more complicated. Right now there seems to be no way to get verified as individual, and only companies will have a chance to get verified: https://github.com/orgs/community/discussions/25265
